frontend/src/static/css/style.scss

```diff
@@ -292,13 +292,21 @@ header{
 
         .title{
             margin-top:1rem;
-            cursor:pointer;
             font-size:1.25rem;
             font-weight:bold;
 
+            .path{
+                cursor:pointer;
+            }
+
             .loc{
                 color:mui-color('grey');
             }
+
+            .button{
+                color:mui-color("grey");
+                text-decoration:none;
```

Comment:
minor thing, but add abit of margin left here

Reply:
and maybe down the font size a little? seems a little tacky now that its pretty huge